tasks/reverse_proxy.yml

```diff
@@ -1,45 +1,59 @@
 ---
+- name: remove deprecated nginx
+  docker_container:
+    name: "hub_nginx"
```

Comment:
will not always work, since the actual prefix depends on another setting, see https://github.com/Innoactive/ansible-innoactive-hub-role/blob/d22337abf7915a7cb986ab859c4c0f73a18eac6a/defaults/main.yml#L195